### PR TITLE
Add Form validation

### PR DESCRIPTION
**Pull request by SaboorSohaib**
# The following requirements are implemented

- The content of the email field has to be in lowercase.
- When the user submits the form, you check if the email is in lowercase.
- If the validation is OK, the form is sent.
- If the validation is not OK, you show an error message to the user near the submit button informing them of the error, and the 
  form is not sent

Kindly check my pull requests if there is anything wrong.

### CODE REVIEW

File: form-validation.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    }
+    return false;
+};
+submitBtn.addEventListener('click', (event) => {
+    const validatedEmail = document.getElementById('mail').value;
+    if(!validEmail(validatedEmail)){
+        document.querySelector('#mail-message').innerHTML= "Please type the email in lowercase";
+        event.preventDefault();
+    }
+})
```
Comment (mahbubul14) [inline form-validation.js]: - [ ] If a user submits the contact form with the email field containing only lowercase characters the form should send normally. Currently, it shows the same error message if the email is lowercase or not.

contact

File: form-validation.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (mahbubul14) [inline form-validation.js]: - [ ] Kindly remove the unnecessary blank lines from your code. You should not add blank lines, spaces, or indentations without a reason to make sure the [HTML & CSS best practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md) :+1:

File: style.css [hunk 3]
```diff
@@ -929,6 +929,11 @@
     margin-bottom: -55px;
   }
 
+  #mail-message {
+    font-size: 16px;
+    color: rgb(66, 82, 110);
+  }
+
```
Comment (mahbubul14) [inline style.css:936]: https://github.com/SaboorSohaib/Protfilo/blob/2a7c0316933ea90b9fa6900b214f2cda672190e9/style.css#L1-L6

- [ ] **[Optional:]** Disable linter is not a good practice. Kindly fix it. :+1:

**Review by mahbubul14 — CHANGES_REQUESTED** [review]
Hi @SaboorSohaib ,

Good job so far! :+1: 
There are some issues that you still need to work on to go to the next project but you are almost there!

# Required Changes ♻️

- [ ] Kindly check the inline comments under the review. :arrow_down: 

###  To Highlights
:heavy_check_mark: No linter error.
:heavy_check_mark: Clean code.
:heavy_check_mark: Descriptive PR title & description.

### Optional suggestions

Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.

- [ ] Kindly check the inline comments under the review. :arrow_down: 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification. You can also reach me on [slack](https://microverse-students.slack.com/team/U01J39GL614)

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by sumairq — APPROVED** [review]
### Hi @SaboorSohaib , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Github flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**